lib/puppet/provider/consul_acl/default.rb

```diff
@@ -23,14 +16,14 @@ def self.list_resources
     uri = URI('http://localhost:8500/v1/acl')
     http = Net::HTTP.new(uri.host, uri.port)
 
-    path=uri.request_uri + '/list'
+    path=uri.request_uri + "/list?token=#{acl_api_token}"
```

Comment:
Does this work when the token is set to an empty string? Like consul tolerate a `/list?token=` ?